content/latest/api/ysql/create_sequence.md

```diff
@@ -0,0 +1,167 @@
+- `maxvalue` is the maximum value allowed in the sequence. If this value is reached, `nextval()` will return an error. If `NO MAXVALUE` is specified, the default will be used. Default is 2^63 - 1.
+- `start` is the first value in the sequence. Default is 1.
+- `cache` how many numbers from the sequence to cache in the client. Default is 1.
+
+## Semantics
+- An error is raised if a sequence with that name already exists in the current schema.
+
+## Cache
+In YSQL as in Postgres, the sequence's data is stored in a persistent system table. In YSQL this table has one row per sequence and it stores the sequence data in two values:
+
+- `last_val` stores the last value used or the next value to be used.
+- `is_called` stores whether `last_val` has been used. If false, `last_val` is the next value in the sequence. Otherwise, `last_val` + `INCREMENT` is the next one.
+
+By default (when `INCREMENT` is 1), each call to `nextval()` updates `last_val` for that sequence. In YSQL, the table holding the sequence's data is replicated as opposed to being in the local file system. Each update to this table requires two RPCs (and will be optimized to one RPC in the future), In any case, the latency experienced by a call to `nextval()` in YSQL will be significantly higher than the same operation in Postgres. To avoid such performance degradation, we recommend using a cache value with a value large enough. Cached values are stored in the memory of the local node, and retrieving such values avoids any RPCs, so the latency of one cache allocation can be amortized over all the numbers allocated for the cache.
+
+`SERIAL` types create a sequence with a cache with default value of 1. So `SERIAL` types should be avoided, and their equivalent statement should be used.
+Instead of creating a table with a `SERIAL` type like this:
+```
+CREATE TABLE t(k SERIAL)
+```
+You should create a sequence with a large enough cache first, and then set the column that you want to have a serial type to `DEFAULT` to `nextval()` of the sequence.
+```
+CREATE SEQUENCE t_k_seq CACHE 10000;
+CREATE TABLE t(k integer NOT NULL DEFAULT nextval('t_k_seq');
+```
+
+## Examples
+
+Create a simple sequence that increments by 1 every time `nextval()` is called.
+
+
+```sql
+postgres=# CREATE SEQUENCE s;
+```
+```
+CREATE SEQUENCE
+```
+
+Call `nextval()`.
+
+```sql
+postgres=# SELECT nextval('s');
+```
+```
+ nextval
+---------
+       1
+(1 row)
+```
+
+Create a sequence with a cache of 10,000 values.
+
+
+```sql
+postgres=# CREATE SEQUENCE s2 CACHE 10000;
+```
+```
+CREATE SEQUENCE
+```
+
+In the same session, select `nextval()`.
+```sql
+SELECT nextval('s2');
+```
+```
+ nextval
+---------
+       1
+(1 row)
+```
+
+In a different session, select `nextval()`.
+```sql
+SELECT nextval('s2');
+```
+```
+nextval
+---------
+   10001
+(1 row)
+```
+
+Create a sequence that starts at 0. MINVALUE also has to be changed from its default 1 to something less than or equal to 0.
```

Comment:
This should also be mentioned above where you are describing MINVALUE and START.